using System;

namespace Functional.Tests.Core.Exceptions
{
    public class TestException : Exception
    {
    }
}